### DISCUSSION TITLE
Doc enhancement: ":latest-arm64" is missing?

### DISCUSSION THREAD

**Discussion by RikkiBC** [nextcloud/all-in-one#4019] [Questions] [answered]
### Is your feature request related to a problem? Please describe. 

I went to link someone to the docs to show that ARM64 architecture is supported along with how to use it, but found that there is no more reference to ":latest-arm64" on the front page docs anymore. It used to be there? It must have been removed at some point?

### Describe the solution you'd like

Add it back into the docs please.

### Describe alternatives you've considered

Having to explain it to people myself instead of being able to point to the authoritative docs.

**Comment by Zoey2936** [comment, marked as answer]
you should pull :latest, it supports both arm64 and amd64

**Comment by RikkiBC** [comment]
> you should pull :latest, it supports both arm64 and amd64

Ohhhh, rightio then. Out of curiosity, when did that change? I had to do it the other way when I set up my AIO instance around 15-ish months ago hah.

**Comment by Zoey2936** [comment]
a bit less then a year ago

**Comment by RikkiBC** [comment]
> a bit less then a year ago

Hah I just missed it then.

Do I need to change anything about my AIO container then, given it was set up for ARM the old way? Or will it keep happily spinning and updating until I need to change something?
